orttraining/orttraining/core/framework/ortmodule_graph_builder.h

```diff
@@ -28,6 +29,7 @@ struct OrtModuleGraphBuilderConfiguration {
   bool build_gradient_graph = true;
 
   // TODO: add GraphTransformerConfiguration
```

Comment:
todo is done :)

Reply:
Yes. this comment slipped through conflict resolution.